client/reader/post-time/index.jsx

```diff
@@ -1,54 +1,49 @@
 /**
  * External dependencies
  */
-var debug = require( 'debug' )( 'calyso:reader:post-time' ),
-	React = require( 'react' ),
-	PureRenderMixin = require( 'react-pure-render/mixin' );
-
+import React from 'react';
+import { PureComponent } from 'react';
+import moment from 'moment';
 /**
  * Internal dependencies
  */
-const smartSetState = require( 'lib/react-smart-set-state' ),
-	ticker = require( 'lib/ticker' ),
-	humanDate = require( 'lib/human-date' );
-
-var PostTime = React.createClass( {
+import smartSetState from 'lib/react-smart-set-state';
+import ticker from 'lib/ticker';
+import humanDate from 'lib/human-date';
 
-	mixins: [ PureRenderMixin ],
+export default class PostTime extends PureComponent {
+	smartSetState = smartSetState;
 
-	componentWillMount: function() {
+	componentWillMount() {
 		this._update();
-	},
+	}
 
-	componentDidMount: function() {
-		debug( 'listening for ticks' );
+	componentDidMount() {
 		ticker.on( 'tick', this._update );
-	},
+	}
 
-	componentWillReceiveProps: function( nextProps ) {
+	componentWillReceiveProps( nextProps ) {
 		this._update( nextProps.date );
-	},
+	}
 
-	componentWillUnmount: function() {
+	componentWillUnmount() {
 		ticker.off( 'tick', this._update );
-	},
-
-	smartSetState: smartSetState,
+	}
 
-	_update: function( date ) {
+	_update = date => {
```

Comment:
Can we drop the underscore on the method name?

[Coding guidelines](https://github.com/Automattic/wp-calypso/blob/master/docs/coding-guidelines/javascript.md) say:

> Avoid prefixing method names with _.